ci_framework/roles/tempest/tasks/main.yml

```diff
@@ -37,8 +37,20 @@
     cmd: "podman unshare chown 42480:42480 -R {{ cifmw_tempest_artifacts_basedir }}"
   when: not cifmw_tempest_dry_run | bool
 
-# > {{ cifmw_tempest_artifacts_basedir }}/tempest_output.log"
 - name: Run tempest
-  ansible.builtin.command:
-    cmd: "podman run -v {{ cifmw_tempest_artifacts_basedir }}/:/var/lib/tempest/external_files:Z {{ cifmw_tempest_image }}:{{ cifmw_tempest_image_tag }}"
+  containers.podman.podman_container:
+    name: tempest
+    image: "{{ cifmw_tempest_image }}:{{ cifmw_tempest_image_tag }}"
+    state: started
+    auto_remove: "{{ cifmw_tempest_remove_container | default(false) }}"
+    volume:
+      - "{{ cifmw_tempest_artifacts_basedir }}/:/var/lib/tempest/external_files:Z"
   when: not cifmw_tempest_dry_run | bool
+
+- name: Change tempest directory permission back to original
+  ansible.builtin.file:
+    path: "{{ cifmw_tempest_artifacts_basedir }}"
+    state: directory
+    recurse: true
+    owner: "{{ ansible_user }}"
```

Comment:
you may need to ensure this fact exists - especially in molecule. Not sure we gather_facts in there...